api4/cloud.go

```diff
@@ -111,6 +111,12 @@ func changeSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Log failures for purchase confirmation email, but don't show an error to the user so as not to confuse them
+	// At this point, the upgrade is complete.
+	if nErr := c.App.SendUpgradeConfirmationEmail(); nErr != nil {
+		c.Logger.Error("Error sending purchase confirmation email")
```

Comment:
Is there any additional data we can provide in this error for troubleshooting purposes? Subscriptionid?

Reply:
We could, but since these are installation logs that would all fall under an installation ID (which implies the subscription ID) I'm not sure we'd need it?

Reply:
Gotcha, good point